cmd/test.go

```diff
@@ -30,6 +30,7 @@ var testParams = struct {
 	errLimit     int
 	outputFormat *util.EnumFlag
 	coverage     bool
+	threshold    int
```

Comment:
Why don't we make this a float64, that way we don't have to do a conversion later on?